backend/src/main/resources/schema.graphqls

```diff
@@ -62,13 +62,23 @@ type TestOrder {
   result: String
   dateTested: String
 }
+type User {
+  id: ID
+  firstName: String
+  middleName: String
+  lastName: String
+  suffix: String
+  organization: Organization
+ }
+
 type Query {
   deviceType: [DeviceType]
   patients: [Patient]
-  patient(id: String): Patient
+  patient(id: String!): Patient
```

Comment:
This gets a nice null pointer exception right now otherwise, so making it mandatory seemed polite...